README.md

```diff
@@ -78,38 +78,21 @@ struct A {
 
 ## Releasing
 
-Before you release, make sure CHANGELOG.md is up to date.
+The versions of all public crates in this repository are collectively managed by a single version in the [workspace manifest](https://github.com/near/borsh-rs/blob/master/Cargo.toml).
 
-Use [`cargo-workspaces`](https://github.com/pksunkara/cargo-workspaces) to save time.
+So, to publish a new version of all the crates, you can do so by simply bumping that to the next "patch" version and submit a PR.
 
-### Bump Versions
+We have CI Infrastructure put in place to automate the process of publishing all crates once a version change has merged into master.
 
-```sh
-cargo workspaces version --force 'borsh*' --exact --no-individual-tags patch
-```
-
-This will bump all the versions to the next "patch" release (see `cargo workspaces version --help`
-for more options), create a new commit, push `v0.x.x` tag, push to the master.
-
-### Publish
-
-To publish on crates.io the version that is currently in git:
-
-```sh
-cargo workspaces publish --from-git --skip-published
-```
-
-Alternatively, you may want to combine the version bumping with publishing:
-
-```sh
-cargo workspaces publish --force 'borsh*' --exact --no-individual-tags patch
-```
+However, before you release, make sure the [CHANGELOG](CHANGELOG.md) is up to date.
 
 ### Release on GitHub
 
+After the checks pass and the release to crates.io has succeeded, a new tag will be created on GitHub to match the new version. You can use this tag to manually create a new release on GitHub.
+
 1. Navigate to the [New Release](https://github.com/near/borsh-rs/releases/new) page
-2. Enter the tag name, e.g. `v0.8.0`
-3. Write down the release log (basically, copy-paste from the CHANGELOG)
+2. Select the tag name, e.g. `v0.8.0`
+3. Write down the release log (basically, copy-paste from the [CHANGELOG](CHANGELOG.md))
 4. Publish the release
```

Comment:
Let's just drop this paragraph (and this process): github releases for crates.io crates is a busy work which doesn't add value. Source of truth is crates.io

Reply:
@matklad I find github releases useful if you want to subscribe to releases.

@miraclx Should we automate github releases as well? https://github.com/near/near-cli-rs/blob/3b903daf73afefcc705f10cf09277e8958f73ec8/.github/workflows/release.yml#L11-L37

Reply:
Here's an example of what this looks like when tagged. Except, instead of `crates-0.9.2` it's now `v0.9.2`

![Screenshot_20220124_153502](https://user-images.githubusercontent.com/16881812/150823072-19de454c-7c7a-4de7-9d77-2541689b0ff3.png)

@frol, the action used in the link is currently [unmaintained](https://github.com/actions/create-release), I'm unsure whether it extracts a body by default from the CHANGELOG or we would have to employ a utility for that as well, but I'll look into it. And maybe other maintained alternatives.

Best things about the tag and release on GitHub is it linking back to crates.io as well as clearly showing the commit responsible for said version bump.